base/glibc-compatibility/musl/getauxval.c

```diff
@@ -39,7 +41,9 @@ ssize_t __retry_read(int fd, void * buf, size_t count)
 {
     for (;;)
     {
-        ssize_t ret = read(fd, buf, count);
+        // We cannot use the read syscall as it will be intercept by sanitizers, which aren't
+        // initialized yet. Emit syscall directly.
```

Comment:
Shouldn't the same be done for `close` ?
BTW does it fixes your issue (on non x86_64)?

Reply:
> Shouldn't the same be done for `close`?

Sanitizers likely intercept buffer related syscalls. `open` and `close` don't fit in this category.

> BTW does it fixes your issue (on non x86_64)?

Yes.